src/main/kotlin/study/racingcar/view/OutputView.kt

```diff
@@ -20,18 +22,17 @@ object OutputView {
     }
 
     private fun printCarNameAndDistance(car: Car) {
-        val result = DASHBAR_SYMBOL.repeat(car.currentPosition)
-        car.name
-        println("${car.name} : $result")
+        val result = List(car.currentPosition) { DASHBAR_SYMBOL }.joinToString("")
```

Comment:
자동차마다 List 자료구조를 만들어서 반복시키는건 비효율적인 것 같아요. 
```suggestion
DASHBAR_SYMBOL.repeat(car.currentPosition)
```

이렇게 확장함수를 활용해보는건 어떨까요?